site/docs/components/combo-box/index.mdx

```diff
@@ -26,6 +28,7 @@ data:
       { name: "List", relationship: "contains" },
       { name: "Input", relationship: "contains" },
     ]
+  status: In progress
```

Comment:
Probably something more clear to lab, e.g., "Lab component"

Reply:
in progress doesn't tell direct association to lab component, especially "do not use in production". It's likely different than the "In process" in foundation actual mean

Reply:
@mikearildbrown @origami-z is there a consensus on this? I personally like "in progress" and I think the sidebar icon provides enough context but let me know either way so we can get this merged :)